base/compiler/ssair/inlining.jl

```diff
@@ -1417,11 +1452,14 @@ function late_inline_special_case!(
     params::OptimizationParams)
     params.inlining || return nothing
     (; f, ft, argtypes) = sig
+    isinlining = params.inlining
```

Comment:
In a recent commit I added a change to bail out this case at an earlier point of this function

```suggestion
```